src/styles/landing-page.less

```diff
@@ -19,39 +26,48 @@
     width: 80%;
   }
   .dropdown-menu {
+    margin-top: 0;
     width: 100%;
     // Styles need to be specific enough to override defaults.
     > li > a.catalog-search-match {
       padding-bottom: 5px;
       padding-top: 5px;
     }
+    // important necessary to override defaults
```

Comment:
I'd be more specific: "important needed to override styles declared important in Patternfly"